### PR TITLE
Turn tags summary into markdown

### LINKED ISSUE #350: Auto-resize tag/edit page textareas
**URL**: tag/edit example : https://preprod.cafebabel.com/fr/article/tag/experience/edit/
**Contexte**: je veux ajouter une description à un tag
**Problème**: le `textarea` ne s'agrandit pas lorsque j'ajoute beaucoup de contenu

### PR DESCRIPTION
**Pull request by davidbgk**
fix #350

### CODE REVIEW

**Comment by davidbgk** [comment]
@cafebabel-webmaster assigned to you too as it will require some styling (detail page + edit page autosizing textarea)

**Comment by vinyll** [comment]
So far so good

File: cafebabel/static/css/styles.css [hunk 1]
```diff
@@ -1685,15 +1685,15 @@ select#author {
   margin-bottom: 2rem;
 }
 
-.article-page article p.video-wrapper {
+p.video-wrapper {
```
Comment (vinyll) [inline cafebabel/static/css/styles.css]: Why did you choose to take this tag out of the _article-page_ context? Is it used for the tag summary?

File: cafebabel/templates/articles/tags/detail.html [hunk 2]
```diff
@@ -17,15 +17,15 @@
   {{ meta('property=og:title', self.meta_title()) }}
   {{ meta('property=og:type', 'article') }}
   {% if tag.summary %}
-    {{ meta('property=og:description', tag.summary) }}
+    {{ meta('property=og:description', tag.summary|striptags) }}
```
Comment (vinyll) [inline cafebabel/templates/articles/tags/detail.html:20]: Wondering what the description should be when no summary exist?

File: cafebabel/static/css/styles.css (hunk 1 above)
Comment (johangiraud) [reply cafebabel/static/css/styles.css]: it is.

File: cafebabel/templates/articles/tags/detail.html (hunk 2 above)
Comment (johangiraud) [reply cafebabel/templates/articles/tags/detail.html:20]: nothing?

**Comment by johangiraud** [comment]
Btw it fixes #350

File: cafebabel/templates/articles/tags/edit.html [hunk 3]
```diff
@@ -15,7 +15,7 @@
     <h1>{{ tag.name }}</h1>
     <div class=textarea>
       <label for=summary>Description</label>
-      <textarea id=summary name=summary>{{ tag.summary }}</textarea>
+      <textarea id=summary name=summary>{{ tag.summary or '' }}</textarea>
```
Comment (davidbgk) [inline cafebabel/templates/articles/tags/edit.html:18]: Do not forget to add the `markdowntext` class, see: https://github.com/cafebabel/cafebabel.com/pull/393#pullrequestreview-109376522

File: cafebabel/templates/articles/tags/detail.html (hunk 2 above)
Comment (davidbgk) [reply cafebabel/templates/articles/tags/detail.html:20]: To me there is the test and that's OK not to have a description.

File: cafebabel/templates/articles/tags/edit.html (hunk 3 above)
Comment (johangiraud) [reply cafebabel/templates/articles/tags/edit.html:18]: I removed markdown-helper for tag/edit. Editor's don't need it this place (= easier)

**Review by johangiraud — APPROVED** [review]

**Review by vinyll — APPROVED** [review]

**Pull request merged**